books/free-programming-books-langs.md

```diff
@@ -1271,6 +1272,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [The Complete Redux Book](https://leanpub.com/redux-book) - Boris Dinkevich and Ilya Gelman  *(Leanpub account or valid email requested)*
 
 
+#### Remix
+
+* [Jokes App Tutorial](https://remix.run/docs/en/v1/tutorials/jokes)
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Reply:
added another resource